pkg/controller/controller_clusterservicebroker_test.go

```diff
@@ -232,6 +232,22 @@ func TestShouldReconcileClusterServiceBroker(t *testing.T) {
 	}
 }
 
+// TestReconcileClusterServiceBrokerSetOSBTimeOut
+// verifies that timeout of any request to the
+// broker takes effect.
+func TestReconcileClusterServiceBrokerSetOSBTimeOut(t *testing.T) {
+	const testOSBTimeOut = time.Second * 80
+	_, _, _, testController, _ := newTestController(t, getTestCatalogConfig())
+	testController.OSBAPITimeOut = testOSBTimeOut
+	if err := reconcileClusterServiceBroker(t, testController, getTestClusterServiceBroker()); err != nil {
+		t.Fatalf("This should not fail : %v", err)
+	}
+	createdClient := testController.brokerClientManager.clients[NewClusterServiceBrokerKey(getTestClusterServiceBroker().Name)]
+	if createdClient.clientConfig.TimeoutSeconds != int(testController.OSBAPITimeOut.Seconds()) {
```

Comment:
shouldn't this match the constant on 239? both client.config/timeoutseconds and testcontroller.osbapitimeout.seconds?

Reply:
No, it should not be the same. It is just a test case. Thanks for review it.

Reply:
you set them to be the same on line 241. why bother having the const then?